subiquity/common/filesystem/manipulator.py

```diff
@@ -121,15 +121,24 @@ def create_partition(self, device, gap, spec: FileSystemSpec, **kw):
         self.create_filesystem(part, spec)
         return part
 
-    def delete_partition(self, part, override_preserve=False, allow_renumbering=True):
+    def delete_partition(
+        self,
+        part,
+        override_preserve=False,
```

Comment:
suggestion: maybe that one should be keyword-only as well.

Reply:
+1